pkg/client/mapiserver/volume_stats.go

```diff
@@ -0,0 +1,23 @@
+package mapiserver
```

Comment:
Need to a understand a bit more.
Does this apply to all kinds of storage engines supported by maya?
Can this code reside at _endpoint.go file only.